### PR TITLE
Browser bugs

### PR DESCRIPTION
**Pull request by AlbertoElias**
Fixes issues raised by @andygout #148 #150 #151 #152 

Still missing #154 but that isn't a blocker for Next implementation

### CODE REVIEW

File: src/scss/_edition-switch.scss [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	font-size: 13px;
 	line-height: 1em;
 	top: oGridGutter(M) - 10;
-	left: oGridGutter(M);
-	z-index: 10;
+	// left: oGridGutter(M);
```
Comment (onishiweb) [inline src/scss/_edition-switch.scss]: any reason for this being commented out and not removed?

**Comment by onishiweb** [comment]
One comment, but otherwise looks good

File: src/scss/_edition-switch.scss (hunk 1 above)
Comment (AlbertoElias) [inline src/scss/_edition-switch.scss]: me being forgetful, cheers!

**Comment by onishiweb** [comment]
:+1:

**Pull request merged**